CRM/Contact/Form/Task/EmailCommon.php

```diff
@@ -54,20 +54,13 @@ public static function preProcessFromAddress(&$form, $bounce = TRUE) {
     $form->_emails = $fromEmailValues;
     $defaults = [];
     $form->_fromEmails = $fromEmailValues;
+    if (is_numeric(key($form->_fromEmails))) {
+      $emailID = (int) key($form->_fromEmails);
+      $defaults = CRM_Core_BAO_Email::getEmailSignatureDefaults($emailID);
+    }
```

Comment:
Moving this up seems like it would fix resetting defaults where it seems the intent was to default to the logged in contact, although it seems to do that before anyway so 🤷 

This PR seems ok but I am seeing a weird thing on my local so just want to check that out.

Reply:
@demeritcowboy ueah I read the setting & then potentially usetting of defaults as a mistake - maybe I'm wrong?

Reply:
Wow where did the last 12 hours go? The patch is stale now - EmailCommon doesn't git-apply.

Otherwise if I make those changes manually it seems fine. I'll put up a PR for the thing I was seeing.

Reply:
@demeritcowboy should I rebase or will this be superceded & perhaps closable with your patch?

Reply:
Sorry it should be rebased - my thing ended up being unrelated.

Reply:
ok - rebased

Reply:
thanks - merge-on-pass'd

Reply:
cool